### ISSUE TITLE
[Suggestion] Mine Ore, nearby stone turns into Fractured Stone instead of Cobblestone

### ISSUE THREAD

**Issue by MechanosG** [WhiskyTangoFawks/Expedition#128]
I made a suggestion similar to this once before last year sometime, but since then I've noticed that fractured stone now acts more like an actual block.  It can be placed by the player, there's fractured variants of other blocks too, etc etc.

Anyway, to the point.  I currently have it setup so when a bunch of modded ores are broken, they fracture into cobblestone.  But what I'd like to attempt, is for it to fracture first into fractured stone.  This would give players a bit of forewarning before the roof falls into their face, while also setting up possible chain reactions.  I attempted to do this manually by settings the fracture pairing for stone to the cracked stone instead of cobble.  Which is like..  meta 3 or something?  Anyway, it resulted in a null pointer error.  You told me once that this kinda thing wasn't intended, and I understand that.  That's why I'm suggesting it.  Sounds fun and might help some players rage a little less (they see surrounding stone crack and think "oops, I better step back before I mine more").

**Comment by WhiskyTangoFawks** [comment]
So, just to clarify, instead fracturing into cobblestone, you want ores to fracture adjacent blocks into cracked stone?

**Comment by MechanosG** [comment]
Yes, that's what I'm attempting to do.  Mine an ore, nearby stone cracks but doesn't instantly crumble to cobblestone.  Mine another ore, or mine one of the cracked stone though, and suddenly collapse.  I honestly have no idea if this is a good idea or not.  But I like cave-in mechanics and wanted to toy around with it a bit and see how it feels.
